addons/crm/views/assets.xml

```diff
@@ -2,6 +2,7 @@
 <odoo>
     <template id="assets_backend" name="CRM assets backend" inherit_id="web.assets_backend">
         <xpath expr="." position="inside">
+            <link rel="stylesheet" type="text/scss" href="/crm/static/src/scss/crm.scss"/>
```

Comment:
We generally try to separate css from js templates (like ``<xpath expr="//link[last()]" position="after">`` or ``<xpath expr="script[last()]" position="after">``). Could be great to update code here to match that convention.